examples/3d/solari.rs

```diff
@@ -68,7 +68,7 @@ fn setup(
             SceneRoot(
                 asset_server.load(
                     GltfAssetLabel::Scene(0)
-                        .from_asset("models/PicaPica/pica_pica_-_mini_diorama_01.glb"),
+                        .from_asset("https://github.com/bevyengine/bevy_asset_files/raw/refs/heads/main/picka_pica/mini_dorama_01.glb")
```

Comment:
```suggestion
                        .from_asset("https://github.com/bevyengine/bevy_asset_files/blob/2a5950295a8b6d9d051d59c0df69e87abcda58c3/pica_pica/mini_diorama_01.glb")
```
Fixed typos + use a fixed url